### ISSUE TITLE
The way that the IIndexPopulator is registered is incorrect and causes issues when the Umbraco PDF package is installed

### ISSUE THREAD

**Issue by Shazwazza** [umbraco/Umbraco.Forms.Issues#535]
Both of these projects: Forms + Umbraco.PDF are registering the IIndexPopulators incorrectly and this is unfortunately due to the way that LightInject works.

This should not be done:

```cs
composition.Register<IIndexPopulator, FormsIndexPopulator>(Lifetime.Singleton);
```

because that will just get replaced by anything else that registers IIndexPopulator like this. Instead FormsIndexPopulator will need to be made public and then just registered like:

```cs
composition.Register<FormsIndexPopulator>(Lifetime.Singleton);
```

And light inject will infer this registration and include that registration whenever `IEnumerable<IIndexPopulator>` is injected.

https://github.com/umbraco/Forms/blob/d94f2b40be9df95938a8dff972637807b2a8ae26/src/Umbraco.Forms.Core/Components/UmbracoFormsComposer.cs#L66

This same fix will need to be applied to the PDF project too. Right now, if both of these packages are installed, and depending on how the runtime orders the bootup, it could mean that Forms is left without a populator for its index.

**Comment by AndyButland** [comment]
Thanks @Shazwazza - noted.  I'll get this fixed in the next patch release.

**Comment by AndyButland** [comment]
Resolved in the linked PR and due in 8.7.2,
